azure_functions_worker/dispatcher.py

```diff
@@ -609,6 +609,10 @@ def index_functions(self, function_path: str):
                     len(indexed_functions))
 
         if indexed_functions:
+            fx_metadata_results = loader.process_indexed_function(
+                self._functions,
+                indexed_functions)
+
             indexed_function_logs: List[str] = []
```

Comment:
If theres an error in `index_function_app` vs `process_indexed_function` - how do we differentiate? Ideally, this log should be in a finally or part of the except or something that would work and help us and the Cx differentiate.